docs/getting_started.rst

```diff
@@ -114,7 +114,53 @@ Suppose we have a transcript in CSV format containing students' basic info and t
 |     202    |     Nick   |   Young   |    Male   |  Physics  |    3.6    |
 +------------+------------+-----------+-----------+-----------+-----------+
 
-Firstly in order to set up a table, we need to specify the schema of this transcript.
+When we create a CSV file, we will also need a separate CSV config JSON file.
+
+First, however, we will create a working directory called ``getting-started`` (in this example, it is on ``Desktop``), and create two additional directories within ``getting-started`` called ``data``
+and ``config``.
+
+Note that we can create a variable for the working directory called ``WORKING_DIR``.
+
+.. code-block:: none
+
+  $ WORKING_DIR=/Users/host1/Desktop/getting-started
+  $ cd $WORKING_DIR
```

Comment:
order needs to be changed

1. mkdir getting-started
2. WORKING_DIR = /path/to/getting_started
3. cd $WORKING_DIR
4. mkdir data
5. mkdir config

and It would be great if we can add directory tree at somewhere

```
|-- getting-started
      |-- data
             |-- test.csv
      |-- config
             |-- csv-record-reader-config.json
             |-- transcript-schema.json
             |-- transcript-table-config.json
```